src/tensors/adjoint.jl

```diff
@@ -6,100 +6,38 @@
 Specific subtype of [`AbstractTensorMap`](@ref) that is a lazy wrapper for representing the
 adjoint of an instance of [`AbstractTensorMap`](@ref).
 """
-struct AdjointTensorMap{T,S,N₁,N₂,TT<:AbstractTensorMap{T,S,N₂,N₁}} <:
+struct AdjointTensorMap{T,S,N₁,N₂,TT<:AbstractTensorMap{T,S,N₁,N₂}} <:
```

Comment:
Is this correct? Might be a typo?

Reply:
Yes, definitely is. Result from me first changing the whole adjoint implementation (kind of redoing the work that you already did), and then only afterwards rebasing on master that included your work.